stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -123,8 +126,8 @@ fun TextFieldSection(
  * attribute of [textFieldController] is also taken into account to decide if the UI should be
  * enabled.
  */
-@OptIn(ExperimentalComposeUiApi::class)
 @Composable
+@Suppress("LongMethod")
```

Comment:
Suppressing this for now. All the code in this composable I think is pretty exposed at this level than elsewhere.